frontend/src/organisation/OrganisationDetail.vue

```diff
@@ -2,12 +2,11 @@
   <div class="card">
     <div class="card-body">
       <h4 class="card-title">
-        <icon name="users" /> {{orgUnit.name}}
+        <icon name="users" /> {{orgUnitInfo.name}}
```

Comment:
This renaming seems a bit arbitrary — from the perspective of the frontend, that object is indeed the unit.